augur/routes/contributor_reports.py

```diff
@@ -931,10 +946,16 @@ def new_contributors_stacked_bar():
 
     @server.app.route('/{}/contributor_reports/returning_contributors_pie_chart/'.format(server.api_version),
                       methods=["GET"])
-    def returning_contributor_pie_chart():
```

Comment:
is this simply a convention change, or is It fixing an error. LGTM, I just want to make sure I understand the change. 

Does it change the name of the existing endpoint? If so, @Ulincsys is relying on the current names and we probably need to also provide a legacy alias.

Reply:
This is simply a change to the method name to match the API endpoint path. It will not change the existing endpoint.